frontend/src/App/Menu.js

```diff
@@ -117,7 +117,7 @@ export function Menu({ onChangeCharacter, theme, setTheme, sticker, setSticker }
           <NavBar.Header>
             <MobileNavButton isOpen={isOpenMobileView} setIsOpen={setOpenMobileView} />
             <NavBar.LogoLink to="/">
-              <NavBar.Logo src={logoImage} alt="Contingency Incursions" />
+              <NavBar.Logo src={theme == "Light" ? logoLightImage : logoDarkImage} alt="Contingency Incursions" />
```

Comment:
Check CSS/Theme rules to ensure invert() is not used

Reply:
Already done